src/sass/style.sass

```diff
@@ -133,6 +142,9 @@ h4
         text-decoration: none
         color: #0099ff
 
+    h2
+        margin: 30px 0 15px 0
```

Comment:
This will affect all `h2` titles that are inside a `.content` class.
Is this expected (although I guess if I hadn't even written an `h2` role above, we are not using any such titles for now), and is this robust against future usage?